interactions/api/error.py

```diff
@@ -261,11 +261,112 @@ def lookup() -> dict:
             20002: "Only bots can use this endpoint.",
             20009: "Explicit content cannot be sent to the desired recipient(s).",
             20012: "You are not authorized to perform this action on this application",
-            20016: "This action cannot be performed due to slowmode rate limit.",
+            20016: "This action cannot be performed due to slow-mode rate limit.",
             20018: "Only the owner of this account can perform this action",
             20022: "This message cannot be edited due to announcement rate limits.",
             20028: "The channel you are writing has hit the write rate limit",
-            20031: "Your Stage topic, server name, server description, or channel names contain words that are not allowed",
-            20035: "Guild premium subscription level too low"
-            #  TODO: post-v4: finish dict.
+            20031: "Your Stage topic, server name, server description, "
+            "or channel names contain words that are not allowed",
+            20035: "Guild premium subscription level too low",
+            30001: "Maximum number of guilds reached (100)",
+            30002: "Maximum number of friends reached (1000)",
+            30003: "Maximum number of pins reached for the channel (50)",
+            30004: "Maximum number of recipients reached (10)",
+            30005: "Maximum number of guild roles reached (250)",
+            30007: "Maximum number of webhooks reached (10)",
+            30008: "Maximum number of emojis reached",
+            30010: "Maximum number of reactions reached (20)",
+            30013: "Maximum number of guild channels reached (500)",
+            30015: "Maximum number of attachments in a message reached (10)",
+            30016: "Maximum number of invites reached (1000)",
+            30018: "Maximum number of animated emojis reached",
+            30019: "Maximum number of server members reached",
+            30030: "Maximum number of server categories has been reached",
+            30031: "Guild already has a template",
+            30033: "Max number of thread participants has been reached (1000)",
+            30035: "Maximum number of bans for non-guild members have been exceeded",
+            30037: "Maximum number of bans fetches has been reached",
+            30038: "Maximum number of uncompleted guild scheduled events reached (100)",
+            30039: "Maximum number of stickers reached",
+            30040: "Maximum number of prune requests has been reached. Try again later",
+            30042: "Maximum number of guild widget settings updates has been reached. Try again later",
+            30046: "Maximum number of edits to messages older than 1 hour reached. Try again later",
```

Comment:
Why do some of these (the ones starting with "Maximum") have a number specified and some don't?

Reply:
Ddocs, I just copied their info

Reply:
Ddocs, I just copied their info

Reply:
Ok then